src/datepicker/docs/readme.md

```diff
@@ -134,4 +137,4 @@ Depending on datepicker's current mode, the date may reffer either to day, month
  * `Enter`/`Space`: Select date.
  * `Ctrl`+`Up`: Move to an upper mode.
  * `Ctrl`+`Down`: Move to a lower mode.
- * `Esc`: Will close popup, and move focus to the input.
\ No newline at end of file
+ * `Esc`: Will close popup, and move focus to the input.
```

Comment:
My IDE added a new line character. I haven't read anything about new line at the end of the file in your contribution guidelines, shall I remove it?